src/osc/RTOSCListener.cpp

```diff
@@ -0,0 +1,39 @@
+
```

Comment:
This is one of the only pieces of code I think needs significant changes.
I dont want the OSC code to have to know about the internals of RTcmix.
So rather than calling the parsing code directly like this, have your class
contain a callback function member with the signature (int)(const char *, int), which your
class then will call inside this method:

status = *_parserCallback(rtScript, std::strlen(rtScript));

Have a 'setter' for this callback, which RTcmixMain will call and pass it the parser
function.  This way, the OSC code does not have to have ANY knowledge of RTcmix!